libs/positions/src/lib/positions-table.tsx

```diff
@@ -360,7 +360,7 @@ export const PositionsTable = forwardRef<AgGridReact, Props>(
               ? undefined
               : addDecimalsFormatNumber(data.unrealisedPNL, data.decimals)
           }
-          cellRenderer="PriceFlashCell"
+          cellRenderer="PriceCell"
```

Comment:
chore (blocking): You need to add PriceCell to the components prop of the AgGrid component

Reply:
updated